crates/subspace-networking/src/create.rs

```diff
@@ -231,10 +238,17 @@ pub async fn create(config: Config) -> Result<(Node, NodeRunner), CreationError>
             relay_client,
         });
 
+        // must be set along with `max_established_total` to prevent eclipse attack
+        let max_established_incoming = (max_established_total * 3 / 4).max(1);
```

Comment:
Why not having incoming and outgoing separately and then sum them (together with number of reserved peers) to get total? It seems weird to only support one fraction here. Also you can remove `CONNECTED_PEERS_THRESHOLD` if you call them just `out_peers` and `in_peers`, `out_peers` would indicate node's target.

Reply:
Ok, I separated them and removed `CONNECTED_PEERS_THRESHOLD `. However `CONNECTED_PEERS_THRESHOLD` was different from the `out_peers` (min in contrast with max). We will attempt to connect to other peers much more often now.